site/docs/get-started/quickstart.md

```diff
@@ -128,10 +142,10 @@ helm install fybrik fybrik-charts/fybrik -n fybrik-system --wait
 
 [Modules](../concepts/modules.md) are plugins that the control plane deploys whenever required. The [arrow flight module](https://github.com/fybrik/arrow-flight-module) enables reading data through Apache Arrow Flight API. 
 
-Install the latest[^1] release of arrow-flight-module:
+Install the {{ arrowFlightRelease }}[^1] release of arrow-flight-module:
 
 ```bash
-kubectl apply -f https://github.com/fybrik/arrow-flight-module/releases/latest/download/module.yaml -n fybrik-system
+kubectl apply -f https://github.com/fybrik/arrow-flight-module/releases/download/{{ arrowFlightRelease }}/module.yaml -n fybrik-system
```

Comment:
Note that https://github.com/fybrik/arrow-flight-module/releases/download/latest/module.yaml is not a valid link

Reply:
Thanks, will fix it to `https://github.com/fybrik/arrow-flight-module/releases/latest/download/module.yaml`

Reply:
fixed